client/post-editor/test/post-editor.jsx

```diff
@@ -1,91 +1,74 @@
+/** @jest-environment jsdom */
```

Comment:
Wow, a lot of mocks, this test should probably be split. or, can’t we fix this with shallower rendering?